src/main/proto/spine/options.proto

```diff
@@ -668,7 +668,8 @@ extend google.protobuf.ServiceOptions {
 message IfMissingOption {
 
     // The default error message.
-    option (default_message) = "The field `${parent.type}.${field.path}` of the type `${field.type}` must have a non-default value.";
+    option (default_message) = "The field `${parent.type}.${field.path}` of the type"
```

Comment:
Please put the starting half to the next line. This way it would be easier to read together.